src/java_bytecode/java_bytecode_parser.cpp

```diff
@@ -136,10 +138,10 @@ class java_bytecode_parsert:public parsert
     }
   }
 
-  u8 read_bytes(unsigned bytes)
+  u8 read_bytes(size_t bytes)
```

Comment:
Comment for future improvement: It's a bit weird that we don't have any documentation comment blocks for methods defined with body within the class declaration.